core/storage/statistics/gae_models.py

```diff
@@ -775,6 +1031,17 @@ def get_model(cls, exploration_id, exploration_version, state_name,
     @classmethod
     def _get_entity_id(cls, exploration_id, exploration_version, state_name,
                        calculation_id):
+        """Returns entity_id corresponding to the given exploration state.
+
+        Args:
+            exploration_id: str. ID of the exploration currently being played.
+            exploration_version: int. Version of exploration.
+            state_name: str. Name of current state.
+            calculation_id: str. It tells about the type of calculation.
```

Comment:
Just use "ID of the calculation performed"

Reply:
done.